_posts/en/newsletters/2020-01-22-newsletter.md

```diff
@@ -0,0 +1,117 @@
+---
+title: 'Bitcoin Optech Newsletter #81'
+permalink: /en/newsletters/2020/01/22/
+name: 2020-01-22-newsletter
+slug: 2020-01-22-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter requests review of a method for sending payments
```

Comment:
Usually, if one of the action items is to help test a release candidate, that's included in the excerpt. Any reason not to do that for LND 0.9 here?

Reply:
added